packages/circuits/circom/core/non-qv/processMessages.circom

```diff
@@ -412,8 +379,10 @@ template ProcessOneNonQv(stateTreeDepth, voteOptionTreeDepth) {
 
     // 5. Verify that currentVoteWeight exists in the ballot's vote option root
     // at cmdVoteOptionIndex.
-    b <== currentVoteWeight;
-    c <== cmdNewVoteWeight;
+    var voiceCreditAmountValid = SafeGreaterEqThan(252)([
```

Comment:
checking on dev, this is not there and seems like it's not needed, we do the check already inside message valiadtor, if isValid is true then it means that the voice credits amount is valid